docs/libs/svg.rst

```diff
@@ -25,8 +25,25 @@ Example
 
 .. include:: ../examples/libs/svg/index.rst
```

Comment:
Please move this below your example.

Reply:
Let me confirm, do you want to put the following code into the example document? Or move the whole "svg.rst" to the example directory?